src/test/unit/controller/BenefitsController.test.ts

```diff
@@ -46,12 +58,37 @@ describe('Benefits Controller', () => {
   it('should render the have you got a new job page when no longer working radio button is selected', () => {
     const body = { employeeBenefits: YesOrNo.YES, benefitsCharCount: 'Test benefits text' };
     const userCase = { isStillWorking: StillWorking.NO_LONGER_WORKING };
-    const controller = new BenefitsController();
+    const controller = new BenefitsController(mockLogger);
 
     const req = mockRequest({ body, userCase });
     const res = mockResponse();
     controller.post(req, res);
 
     expect(res.redirect).toBeCalledWith(PageUrls.NEW_JOB);
   });
+
+  it('should add the benefits form value to the userCase', () => {
+    const body = { employeeBenefits: YesOrNo.NO };
+
+    const controller = new BenefitsController(mockLogger);
+
+    const req = mockRequest({ body });
+    const res = mockResponse();
+    req.session.userCase = undefined;
+
+    controller.post(req, res);
+
+    expect(req.session.userCase).toStrictEqual({ employeeBenefits: YesOrNo.NO });
+  });
+
+  it('should run logger in catch block', async () => {
```

Comment:
For all these logger tests, its not clear to me why they're hitting the catch block. Do you know?

Reply:
Because the axios.put method fires and error without all provided details(as I debugged it) it you want I can share my screen in debug mode and show what is happening.

Reply:
Okay, I think in future we should actually throw an error in the catch block and test it but for now this will be okay to get sonar coverage